src/tests/mockdata.js

```diff
@@ -0,0 +1,12 @@
+import dotenv from 'dotenv';
```

Comment:
@william000000 to ensure our work is neater we can add this in a mockdata folder since we may have multiple mockdata files for other features.